core/src/com/unciv/ui/screens/civilopediascreen/FormattedLine.kt

```diff
@@ -316,6 +293,30 @@ class FormattedLine (
         return table
     }
 
+    private fun renderExtraImage(labelWidth: Float): Table {
+        val table = Table(BaseScreen.skin)
+        fun getExtraImage(): Image? {
+            if (ImageGetter.imageExists(extraImage))
+                return if (centered) ImageGetter.getDrawable(extraImage).cropToContent()
+                    else ImageGetter.getImage(extraImage)
+            val externalImage = ImageGetter.findExternalImage(extraImage)
+                ?: return null
+            return ImageGetter.getExternalImage(externalImage)
+        }
+        try {
+            val image = getExtraImage() ?: return table
+            // limit larger cordinate to a given max size
+            val maxSize = if (imageSize.isNaN()) labelWidth else imageSize
+            val (width, height) = if (image.width > image.height)
+                maxSize to maxSize * image.height / image.width
+            else maxSize * image.width / image.height to maxSize
+            table.add(image).size(width, height)
+        } catch (exception: Exception) {
```

Comment:
Under what conditions would this throw an exception?! I want to see them!
https://yairm210.github.io/Unciv/Guiding-Principles/#crash-early-crash-often

Reply:
Sounds to me like we should remove the catch and see who's really behind that mask 😈